src/pages/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { useEffect } from "react";
+import { Button, Card, Container, Form, FormGroup } from "react-bootstrap";
+import { useForm, useFormState } from "react-hook-form";
+import { useLocation, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import {
+  capitalLetterPattern,
+  emailpattern,
+  lowerLetterPattern,
+  numbersPattern,
+  specialCharacterPattern,
+} from "../../constants";
+import "./SignUp.scss";
+
+function SignUp() {
+  const location = useLocation();
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [location]);
+
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    control,
+    watch,
+    formState: { errors },
+  } = useForm({
+    mode: "onChange",
+    reValidateMode: "onChange",
+    criteriaMode: "all",
+  });
+
+  const { dirtyFields } = useFormState({
+    control,
+  });
+  const watchPassword = watch("password");
+
+  useEffect(() => {
+    document.title = "Xtreme Cars | Sign Up";
```

Comment:
Good thing you considered to add dynamic title even if its not mentioned in the ticket